virt/virt-2-4-release-notes.adoc

```diff
@@ -8,56 +8,105 @@ toc::[]
-volumeMode set in the `kubevirt-storage-class-default` ConfigMap. This issue has been fixed.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753688[*BZ#1753688*])
-
-* Previously, when navigating to the *Virtual Machines Console* tab,
-sometimes no content was displayed. This issue has been fixed.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753606[*BZ#1753606*])
-
-* Previously, attempting to list all instances of the {VirtProductName} operator
-from a browser resulted in a 404 (page not found) error.
-This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1757526[*BZ#1757526*])
-
-* Previously, if a virtual machine used guaranteed CPUs, it was not scheduled
-because the label `cpumanager=true` was not automatically set on nodes.
-This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1718944[*BZ#1718944*])
+//CNV-5421 - DataVolumes have been enhanced with conditions that make it easier to understand the state of VM disk import, clone, and upload.
 
+[id="virt-2-4-web-new"]
+=== Web console
+//Placeholder for new content.
+
+//CNV-5275 - Update template designs - virtual machines and virtual machine templates merged under one menu called Virtualization.
+
+//CNV-5272 - Configure scheduling directly from CNV UI.
+
+//CNV-5273 - Attach a config map / secret / service account to a VM from the UI.
+
+== Notable technical changes
+//Placeholder for new content.
+
+//CNV-5371 - Import existing virtual machines from Red Hat Virtualization to OpenShift Virtualization.
+
+[id="virt-2-4-known-issues"]
 == Known issues
+* This is work in progress. This content has not been updated.
+
+* KubeMacPool is disabled in {VirtProductName} {VirtVersion}. This means that a secondary
+interface of a Pod or virtual machine obtains a randomly generated MAC address rather
+than a unique one from a pool. Although rare, randomly assigned MAC addresses can conflict.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1816971[*BZ#1816971*])
+
+* Sometimes, when attempting to edit the subscription channel of the *{VirtProductName} Operator* in the web console, clicking the
+*Channel* button of the *Subscription Overview* results in a JavaScript error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1796410[*BZ#1796410*])
+
+** As a workaround, trigger the upgrade process to {VirtProductName} {VirtVersion}
+from the CLI by running the following `oc` patch command:
++
+----
+$ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.3 && oc patch -n "${TARGET_NAMESPACE}" $(oc get subscription -n ${TARGET_NAMESPACE} --no-headers -o name) --type='json' -p='[{"op": "replace", "path": "/spec/channel", "value":"'${CNV_CHANNEL}'"}, {"op": "replace", "path": "/spec/installPlanApproval", "value":"Automatic"}]'
+----
++
+This command points your subscription to upgrade channel `2.3` and enables automatic updates.
 
-* If you have {VirtProductName} 2.1.0 deployed, you must first upgrade {VirtProductName}
```

Comment:
Just making sure that this was removed on purpose (note above says this section is unchanged)

Reply:
Great catch. Must have gotten axed inadvertently during edits. Added it back.

Reply:
I don't have the entire paragraph text. Can you please paste it here if you have it. Thanks.